src/routes/packages.js

```diff
@@ -135,8 +149,7 @@ export async function publishPackage (c) {
   const new_versions = Object.keys(body.versions).filter(v => !versions.includes(v))
 
   if (!results || results.length === 0) {
-    const insertQuery = `INSERT INTO packages (name, tags) VALUES ("${pkg}", json('{"latest": "${new_versions[0]}"}'))`
-    await c.env.DB.prepare(insertQuery).run()
+    await mergeTags(c.env.DB, pkg, { latest: new_versions[0] })
```

Comment:
Since this, somewhat, expands our tagging support, we should validate the tag values before inserting/updating.

Reply:
how could this been done.